apps/crn-server/test/data-providers/contentful/calendars.data-provider.test.ts

```diff
@@ -288,6 +397,74 @@ describe('Calendars data provider', () => {
       expect(calendarMock.update).toHaveBeenCalled();
       expect(calendarMockUpdated.publish).toHaveBeenCalled();
     });
+
+    test('Should update the calendar - with new google api metadata - without previous google api metadata', async () => {
```

Comment:
why does it not set the associated google calendar ID here though? I seem to understand the purpose of the tests down below but in that case I don't know what is going on in this one